tests/yomi/start_qemu.pm

```diff
@@ -0,0 +1,91 @@
+# Yomi's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Install QEMU and launch the inner minion
+# Maintainer: Alberto Planas <aplanas@suse.de>
+
+use strict;
+use warnings;
+use base "consoletest";
+use testapi;
+use utils;
+
+sub install_qemu {
+    zypper_call 'in qemu';
+    zypper_call 'in qemu-x86';
+    zypper_call 'in qemu-tools';
+    zypper_call 'in ovmf';
+}
+
+sub assert_script_run_qemu {
+    my ($command) = @_;
+    assert_script_run "ssh -oStrictHostKeyChecking=no -p 10022 localhost '$command'";
+}
+
+sub start_qemu {
+    my ($scenario) = @_;
+
+    assert_script_run 'qemu-img create -f qcow2 hda.qcow2 24G';
+
+    my $efi = '';
+    if (index($scenario, '-efi') != -1) {
+	$efi = '-drive if=pflash,format=raw,unit=0,readonly,file=/usr/share/qemu/ovmf-x86_64-code.bin ' .
+	    '-drive if=pflash,format=raw,unit=1,file=/usr/share/qemu/ovmf-x86_64-vars.bin ';
+    }
+
+    type_string 'nohup qemu-system-x86_64 ' .
```

Comment:
I think it's better to put this, into a bash script, otherwise this test will end up with a lot of failures due to mistyping and stuff...

Reply:
Sure. From where I can serve the script?

Reply:
put it in [data/yomi/yourscript](https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/master/data), and download it later from the SUT:

`curl -O ' . autoinst_url . "/data/yomi/yourscript"`